packages/session-replay-browser/rollup.config.js

```diff
@@ -1,15 +1,37 @@
 import { iife, umd } from '../../scripts/build/rollup.config';
 
-
 import resolve from '@rollup/plugin-node-resolve';
 import replace from '@rollup/plugin-replace';
 import { terser } from 'rollup-plugin-terser';
 import typescript from '@rollup/plugin-typescript';
 import { rollup } from 'rollup';
 import path from 'node:path';
+import commonjs from '@rollup/plugin-commonjs';
 
 iife.input = umd.input;
 iife.output.name = 'sessionReplay';
+iife.output.inlineDynamicImports = true;
+umd.output.inlineDynamicImports = true;
```

Comment:
> This will inline dynamic imports instead of creating new chunks to create a single bundle. https://rollupjs.org/configuration-options/#output-inlinedynamicimports

umd and iffe outputs are what we host on `https://cdn.amplitude.com/libs/session-replay-browser-xxx-min.js.gz`. Setting true will not help to reduce the size of the CDN file. Ideally, we should have two chunks with https://rollupjs.org/configuration-options/#output-chunkfilenames. One for main SR SDK and the other for console record library.